immut/sorted_map/sorted_map.mbti

```diff
@@ -106,6 +108,7 @@ impl T {
   size[K, V](Self[K, V]) -> Int
   to_array[K, V](Self[K, V]) -> Array[(K, V)]
   to_json[K : Show, V : ToJson](Self[K, V]) -> Json
+  union[K : Compare, V](Self[K, V], Self[K, V]) -> Self[K, V]
```

Comment:
There's unsolved api design issue on `union` of map data structures. Could you please remove `union` implementation in this PR?